csfieldguide/static/interactives/cmy-mixer/js/cmy-mixer.js

```diff
@@ -158,11 +179,26 @@ CMY_Mixer.setColorFromInputBox = function() {
     yellow_val = parseInt(yellow_val, 16);
   }
   var cmy_as_rgb = cmy_to_rgb(cyan_val, magenta_val, yellow_val);
+  
+  if (!useHex) {
+    // Display colour as text in coloured box
+    // There is no equivalent hex representation of colour in CMY
```

Comment:
Do you think we need to state this somewhere on the interactive? Not sure what the best thing to do is.

We could:
- Show the RGB hex code equivalent and explain it
- Show no hex code and explain why
- Leave it as is but then users may get confused as to why it doesn't show anything?

:woman_shrugging: :woman_shrugging: :woman_shrugging:

Reply:
Well since cmy isn't actually used either and the string doesn't show any new information I'm a big fan of not having the string form at all. We should discuss it

Reply:
I think I'd be a fan of not having the string form as well.